app/views/records/_show.html.erb

```diff
@@ -10,7 +10,7 @@
         <b><%= t('biomodelos.visor.selected_record.id_of_the_record') %></b> <span id="record_id"><%= @record["id"] %></span><br> <!-- _id -->
         </p>
       </blockquote>
-      <% if user_signed_in? %>
+      <% if user_signed_in? && !@record["reportedDate"].nil? %>
```

Comment:
The validation will only allow to report and edit an already reported record.

Also notice that in the database the field could not exist, but I don't remember how it's mapped to the model in the front.